README.md

```diff
@@ -32,7 +32,7 @@ For end-users
 ASPIRE is a pip-installable package that works on Linux/Mac/Windows, and requires Python 3.7. The simplest option is to use Anaconda 64-bit for your platform with a minimum of Python 3.7 and `pip`, and then use `pip` to install `aspire` in that environment.
```

Comment:
Should we say requires Python 3.8 if we're encouraging using the `conda` env?

Reply:
I think just the two 3.7's in these two sentences should change the 3.8, but the rest are okay because we are still supporting, and they can use pip with >= 3.7 etc

Reply:
ASPIRE requires >=3.7 for pip.  You are right that this is somewhat misleading. I will change it (similar to installation.rst) Let me know if it works better.

Reply:
(We want it to be accurate, but also keep in mind if we invert the conda/pip version pinning relationship that the docs will need to be updated again anyway).